doc/BROWSER/SCHEMA.md

```diff
@@ -220,14 +217,10 @@ var actionData = {
     last_name: null,
     email: 'tester@exmaple.com',
   },
-  currency: null,
   line_items: [
     {
       variant_id: '34454',
-      quantity: null,
-      price: null,
-      item_id: '34454',
-      total_discount: null
+      item_id: '34454'
     }
   ]
 }
```

Comment:
Hey Amy slight change, can you help me make this amendment?

In "important" section at the end can you change the first point to this:

> - If your app supports "guest" users (eg. guest checkouts) where ID is not generated yet, please use a temporary unique ID for the `user.id` field as it is a required field. In this case, `user.email` will be used to identify the user.